extension/source/background/QuillController.ts

```diff
@@ -151,7 +157,21 @@ export default class QuillController {
 
         const address = await this.keyringController.rpc.addAccount(request);
 
-        this.preferencesController.createUser(address, 'USD', 'light');
+        const network = await this.cells.network.read();
+        const keyring = await this.cells.keyring.read();
+
+        const wallet = keyring.wallets.find(
+          (w) => w.networks[network.networkKey]?.address === address,
+        );
+
+        assert(wallet !== undefined);
+
+        this.preferencesController.createUser(
+          wallet.publicKeyHash,
+          'USD',
+          'light',
```

Comment:
Looks like you can detect this on init, as well as respond to changes if the user switches it in their browser/OS https://stackoverflow.com/a/57795495/648624 #outofscope

Reply:
Oooh sounds like a #good-first-issue.

Reply:
Created: https://github.com/web3well/bls-wallet/issues/302.